conf.py

```diff
@@ -204,47 +246,48 @@
         'versions': [
             {
                 'product': '377',
-                'java': '11',
-                'java_base_image_sha256': '7929833412c331fc23cde0e23ca730d652c0be61a8a69c8a82b2af937a3fbd4e',
+                'java-base': '11',
                 'opa_authorizer': '0.1.0',
             },
             {
                 'product': '387',
-                'java': '11',
-                'java_base_image_sha256': '7929833412c331fc23cde0e23ca730d652c0be61a8a69c8a82b2af937a3fbd4e',
+                'java-base': '11',
                 'opa_authorizer': '0.1.0'
             },
             {
                 'product': '395',
-                'java': '17',
-                'java_base_image_sha256': '2b8d60d1ab50d56240cb6286d6bc377410442afbfc3292d81be5674bc0b51724',
+                'java-base': '17',
                 'opa_authorizer': 'stackable0.1.0'
             },
             {
                 'product': '396',
-                'java': '17',
-                'java_base_image_sha256': '2b8d60d1ab50d56240cb6286d6bc377410442afbfc3292d81be5674bc0b51724',
+                'java-base': '17',
                 'opa_authorizer': 'stackable0.1.0'
             },
             {
                 'product': '403',
-                'java': '17',
-                'java_base_image_sha256': '2b8d60d1ab50d56240cb6286d6bc377410442afbfc3292d81be5674bc0b51724',
+                'java-base': '17',
                 'opa_authorizer': 'stackable0.1.0'
             },
         ],
     },
     {
         'name': 'tools',
-        'versions': [{'product': '0.2.0'}],
-    },
-    {
-        'name': 'testing-tools',
-        'versions': [{'product': '0.1.0'}],
+        'versions': [{'product': '0.2.0', 'stackable-base': '1.0.0'}],
     },
+    # Build is broken
+    # {
+    #     'name': 'testing-tools',
+    #     'versions': [{'product': '0.1.0'}],
+    # },
```

Comment:
This seems to have bitrotted heavily, do we still want this?

Reply:
Oh, I didn't know that. This image is used pretty much everywhere in the kuttl tests.

Reply:
Yeah, the tools Image should not be needed anymore, but the testing-tools is used in tests - obviously - and demos

Reply:
Ok, I've created https://github.com/stackabletech/docker-images/issues/324 for this then